### PR TITLE
Fix inconsistent imagePullSecrets inputs

### PR DESCRIPTION
**Pull request by rappizs**
## Description
Fixes https://github.com/banzaicloud/koperator/issues/999 

## Type of Change

- [x] Bug Fix

## Checklist

- [x] I have read the [contributing guidelines](../CONTRIBUTING.md)
- [x] Existing issues have been referenced (where applicable)
- [x] I have verified this change is not present in other open pull requests
- [x] Functionality is documented
- [x] All code style checks pass
- [x] New code contribution is covered by automated tests
- [x] All new and existing tests pass

### CODE REVIEW

**Review by pregnor — APPROVED** [review]
LGTM, @panyuenlau , @bartam1 please take a look at this.

**Review by bartam1 — DISMISSED** [review]

**Comment by bartam1** [comment]
LGTM but is there any reason why we need both?
I mean add imagePullSecret in deployment and serviceaccount same time.

File: charts/kafka-operator/values.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ replicaCount: 1
 
 # Lists the secrets you need to use to pull kafka-operator image from a private registry.
 imagePullSecrets: []
```
Comment (bartam1) [inline charts/kafka-operator/values.yaml:8]: I think we also can use this here:
imagePullSecrets:
or 
# imagePullSecrets:

This way we dont need to use [] which can be mislead

Comment (rappizs) [reply charts/kafka-operator/values.yaml:8]: Good idea, thanks!

**Comment by pregnor** [comment]
> > LGTM but is there any reason why we need both?
> 
> The Deployment is for the kafka-operator itself and the ServiceAccount is for the Prometheus auth proxy. As far as I understand the Deployment uses a different ServiceAccount. Correct me @pregnor if I'm wrong.

https://github.com/banzaicloud/koperator/blob/7edcd84cef2c267bb8c788c7278e1ab57f55dfc5/charts/kafka-operator/values.yaml#L74
This is the `kafka-operator-authproxy` serviceAccount we are giving access to the images.

The other place for the imagePullSecret is at the Koperator deployment: https://github.com/banzaicloud/koperator/blob/4b4ea64892334f1cd6c8d07f5bcb9f372ab4cbce/charts/kafka-operator/templates/operator-deployment-with-webhook.yaml#L139
But the serviceAccount used there is different: https://github.com/banzaicloud/koperator/blob/4b4ea64892334f1cd6c8d07f5bcb9f372ab4cbce/charts/kafka-operator/templates/operator-deployment-with-webhook.yaml#L143
which is `kafka-operator`: https://github.com/banzaicloud/koperator/blob/4b4ea64892334f1cd6c8d07f5bcb9f372ab4cbce/charts/kafka-operator/values.yaml#L38.

So we need the `imagePullSecrets` definition at both places.

**Review by pregnor — APPROVED** [review]

**Review by bartam1 — APPROVED** [review]

**Pull request merged**